wpilibj/src/test/java/edu/wpi/first/wpilibj/command/ConditionalCommandTest.java

```diff
@@ -75,7 +76,7 @@ void onTrueTest() {
     assertConditionalCommandState(m_command, 1, 5, 5, 1, 0);
 
     assertTrue(m_onTrue.getInitializeCount() > 0, "Did not initialize the true command");
-    assertTrue(m_onFalse.getInitializeCount() == 0, "Initialized the false command");
+    assertSame(m_onFalse.getInitializeCount(), 0, "Initialized the false command");
```

Comment:
These would be better as `assertEquals` so we can see the initialization count, not just that they _were_ initialized